shell/docker-build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Ensure we fail the job if any steps fail
+# Do not set -u as DOCKER_ARGS may be unbound
+set -e -o pipefail
+
+# Switch to the directory where the Dockerfile is
+cd "$DOCKER_ROOT"
+
+# Build the docker image
+
+# Allow word splitting
+# shellcheck disable=SC2086
+docker build $DOCKER_ARGS . | tee "$WORKSPACE/docker_build_log.txt"
+DOCKER_IMAGE=$(grep -Po 'Successfully built \K[^ ]+' \
+    "$WORKSPACE/docker_build_log.txt")
+
+IMAGE_NAME="$DOCKERREGISTRY/$DOCKER_NAME:$DOCKER_TAG"
```

Comment:
Yes, and where I had it in my scripts before I did some re-arranging I actually had a comment as to why. I'll get that in again before I push an updated one with the verify template as well.